sonar-text-dotnet/pipeline_scripts/analysis_begin_step.ps1

```diff
@@ -1,12 +1,12 @@
 Set-Location $env:PROJECT_DIR
 
-if ($env:CIRRUS_PR -eq "") {
+if ($env:CIRRUS_BRANCH -eq "master") {
     Write-Host "Execute analysis begin step for master branch"
 
     SonarScanner.MSBuild.exe begin `
         /k:$env:PROJECT_NAME `
         /n:$env:PROJECT_NAME `
-        /v:$env:CIRRUS_CHANGE_IN_REPO `
+        /v:$env:PROJECT_VERSION `
```

Comment:
When in master I set the project version instead of the commit as it was before.